dev/TeachingTip/TestUI/TeachingTipPage.xaml

```diff
@@ -571,5 +577,29 @@
             </StackPanel>
         </ScrollViewer>
 
+        <Grid Grid.Column="2">
+            <Button x:Name="targetButtonRightEdge" HorizontalAlignment="Right" AutomationProperties.Name="TargetButtonRightEdge" VerticalAlignment="Top" Click="OnShowButtonClickedRightEdge"
+                    Height="40" Width="100">Click to open TeachingTip
+                <Button.Resources>
+                    <muxc:TeachingTip x:Name="TeachingTipInResourcesOnEdge" 
+                                          AutomationProperties.Name="TeachingTipInResources"
```

Comment:
>TeachingTipInResources [](start = 69, length = 22)

This has the same automation name as the other teaching tip on this page, I'd be surprised if this didn't cause test issues

Reply:
Interestingly this did not crash the tests. I guess since we don't have test that actually get the TeachingTips using their UIA name, no test is impacted by conflicting UIA names.